docs/reference/dapis/understand/index.md

```diff
@@ -50,47 +52,25 @@ to get the value of a dAPI.
 ## The role of Airnode
 
 Airnode is a flexible off-chain module that can support multiple protocols. Most
-noticeably are its implementation of the request-response protocol (RRP) and
-data feeds.
+noticeably is its implementation of the request-response protocol (RRP) and data
+feeds.
 
 An Airnode is owned by an API provider and integrates itself along side their
-API operations. Functionality within Airnode monitors one or more API operations
-hosted by an API provider and looks for a preset deviation of a data feed value
-(e.g., plus or minus 1%). When the deviation threshold is reached, Airnode
-pushes the new value on-chain into the beacon store. Each beacon represents a
-value from a single API operation.
+API operations. Airnode interfaces with these operations and gathers its signed
+data at the request of AirSeeker. AirSeeker uses the signed data to determine if
+the deviation of a dAPI value (e.g., plus or minus 0.25%, 0.5%, 1%) warrants an
+on-chain update of its beacon(s).
 
 <img src="../assets/images/beacons-airnode.png" style="width:80%;">
 
-In the diagram above, company ABC has two API operations (B and C) and a single
-Airnode that monitors the API operations. When the deviation threshold is
-reached for either operation it will update the corresponding beacons, in this
-case `1FeexV6A` and `1AC4fMwg`.
+In the diagram above, companies ABC and XYZ both have an API operation (A and B)
+that is used to determine the value of the dAPI ZIL/USD . AirSeeker regularly
```

Comment:
```suggestion
In the diagram above, companies XYZ and ABC both provide ZIL/USD values, A and B, respectively, that are aggregated to determine the dAPI ZIL/USD value. Airseeker regularly
```

Reply:
For all the commas it does read nicely. You should be an English professor. Done